gce/image/build_image.sh

```diff
@@ -26,6 +27,7 @@ print_usage() {
     echo "  --product            scylla or scylla-enterprise"
     echo "  --download-no-server download all rpms needed excluding scylla using .repo provided in --repo-for-install"
     echo "  --dry-run            validate template only (image is not built)"
+    echo "  --build-id           jenkins job build ID, will be use for image name"
```

Comment:
Please fix the description not to refer to Jenkins build ID.

Reply:
Changed